packages/patternfly-4/react-core/src/components/Alert/helpers/AlertBody.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly/components/Alert/alert.css';
+import { Button, ButtonVariant } from '../../Button';
+import { TimesIcon } from '@patternfly/react-icons';
+import { Omit } from '../../../helpers/typeUtils';
+
+export interface AlertBodyProps extends Omit<React.HTMLProps<HTMLDivElement>, 'title'> {
+  /** title */
+  title?: React.ReactNode;
+  /** children */
+  children?: React.ReactNode;
+  /** className */
+  className?: string;
+  /** onClose */
+  onClose?: () => void;
+  /** closeButtonAriaLabel */
+  closeButtonAriaLabel?: string;
+};
+
+export const AlertBody: React.FunctionComponent<AlertBodyProps> = ({
+  title = null,
+  className = '',
+  children = '',
+  onClose = (): any => undefined,
```

Comment:
Thinks you can change this too:
```suggestion
  onClose,
```